presidio-analyzer/tests/test_spacy_recognizer.py

```diff
@@ -38,7 +38,7 @@ def prepare_and_analyze(nlp, recognizer, text, ents):
         ("I bought my car in May", 1, ((19, 22),), 1),
         ("May 1st", 1, ((0, 7),), 1),
         ("May 1st, 1977", 1, ((0, 13),), 1),
-        ("I bought my car on May 1st, 1977", 1, ((19, 32),), 1),
+        ("I bought my car on May 1st, 1977", 2, ((19, 26), (28, 32)), 1),
```

Comment:
spacy now recognizes it in two parts and recognizes 1977 as a date, unfortunately. Our recognizer, which is on by default should return better results and prioritized in anonymizer normalization

Reply:
spaCy gets priority because it has a higher confidence value?

Reply:
I think these tests only use spacy recognizer. I mentioned that our recogznizer will possibly be prioritized over spacy's because of this split.